lib/wisper/registration/object.rb

```diff
@@ -22,7 +22,22 @@ def broadcast(event, publisher, *args)
     private
 
     def publisher_in_scope?(publisher)
-      allowed_classes.empty? || publisher.class.ancestors.any? { |ancestor| allowed_classes.include?(ancestor.to_s) }
+      constantized_classes.empty? || publisher.class.ancestors.any? { |ancestor| constantized_classes.include?(ancestor) }
```

Comment:
Instead of looping with `any?`, the `Array` intersection operator `&` can be used.  It is much faster.

    constantized_classes.empty? || (publisher.class.ancestors & constantized_classes).present?